incubating/codecov-reporter/step.yaml

```diff
@@ -57,13 +57,18 @@ spec:
         "additionalProperties": false,
         "patterns": [],
         "required": [
-            "codecov_integration"
+            "codecov_integration",
+            "os"
```

Comment:
It is breaking change, will cause problem with existing customers, I think we should add at least some default value